docs/source/setup/custom-startup.rst

```diff
@@ -59,6 +61,49 @@ if it is on the path) when we start the scheduler::
 
    dask-scheduler --preload scheduler-setup.py --print-count
 
+Types
+~~~~~
+
+Preloads can be specified as any of the following forms:
+
+-   A path to a script, like ``/path/to/myfile.py``
+-   A module name that is on the path, like ``my_module.initialize``
+-   The text of a Python script, like ``import os; os.environ["A"] = "value"``
+
+Configuration
+~~~~~~~~~~~~~
+
+Preloads can also be registered with configuration at the following values:
+
+.. code-block:: yaml
+
+   distributed:
+     scheduler:
+       preload:
+       - "import os; os.environ['A'] = 'b'"  # use Python text
+       - /path/to/myfile.py                  # or a filename
+       - my_module                           # or a module name
+       preload_argv:
+       - []                                  # Pass optional keywords
```

Comment:
I assume these are per preload script (so only `myfile.py` has args?).

It looks like this was an existing pattern, but I find it a bit hard to read (need to mentally align the script with the args). The following might have been cleaner:

```yaml
distributed:
  scheduler:
    preloads:
      - script: "import os; os.environ['A'] = 'b'"
      - script: /path/to/myfile.py
        argv: ["--option", "value"]
      - script: my_module
```

Reply:
I entirely agree.  I probably won't make this change though.  A few people asked me about preloads in one day, so I figured it would make sense to document them a bit better.  I'm entirely in favor of changing how we spell these out in configuration, but I don't think that that should block this PR.

Reply:
Yeah, if it's an existing pattern then this is a separate issue. I wasn't sure if this config just went in or has existed for a while.